src/main/webapp/app/exercises/programming/manage/grading/feedback-analysis/feedback-analysis.component.html

```diff
@@ -16,20 +16,29 @@
 </ng-template>
 <div class="m-3">
     <div class="row m-0 justify-content-between align-items-center">
-        <h3 class="m-0 col p-0" [jhiTranslate]="TRANSLATION_BASE + '.title'" [translateValues]="{ exerciseTitle: exerciseTitle() }"></h3>
-        <div class="col-md-auto d-flex align-items-center position-relative">
-            <button class="btn me-2" (click)="openFilterModal()" [ngClass]="{ 'btn-secondary': selectedFiltersCount() == 0, 'btn-success': selectedFiltersCount() != 0 }">
-                <fa-icon [icon]="faFilter"></fa-icon>
-                <span [jhiTranslate]="TRANSLATION_BASE + '.filter'"></span> ({{ selectedFiltersCount() }})
-            </button>
-            <input
-                type="search"
-                class="form-control"
-                [placeholder]="TRANSLATION_BASE + '.search' | artemisTranslate"
-                aria-label="Search feedback"
-                [ngModel]="searchTerm()"
-                (ngModelChange)="search($event)"
-            />
+        <div class="justify-content-between d-flex align-items-center">
+            <h3 class="m-0 p-0" [jhiTranslate]="TRANSLATION_BASE + '.title'" [translateValues]="{ exerciseTitle: exerciseTitle() }"></h3>
+            <div class="d-flex align-items-center">
+                <div class="form-switch d-flex align-items-center flex-md-shrink-0 me-2">
+                    <input class="form-check-input mt-0" type="checkbox" (click)="toggleLevenshtein()" [checked]="levenshtein()" style="width: 2.5rem; height: 1.25rem" />
+                    <label class="ms-1 me-2 mb-0">
+                        <span [jhiTranslate]="TRANSLATION_BASE + '.groupFeedback'"></span>
+                        <fa-icon size="sm" [icon]="faCircleQuestion" [ngbTooltip]="TRANSLATION_BASE + '.groupFeedbackTooltip' | artemisTranslate" class="btn-warning"></fa-icon>
+                    </label>
+                </div>
+                <button class="btn me-2" (click)="openFilterModal()" [ngClass]="{ 'btn-secondary': selectedFiltersCount() == 0, 'btn-success': selectedFiltersCount() != 0 }">
```

Comment:
You could write the ngClass this way: `[ngClass]=" selectedFiltersCount() ? 'btn-success' :  'btn-secondary'"`